src/Primitives.hs

```diff
@@ -286,6 +286,12 @@ primitiveInfo _ ctx [target@(XObj (Sym path@(SymPath _ name) _) _ _)] = do
           case Meta.get "doc" metaData of
             Just (XObj (Str val) _ _) -> liftIO $ putStrLn ("Documentation: " ++ val)
             Nothing -> return ()
+          case Meta.get "implements" metaData of
+            Just xobj@(XObj object info _) -> do
+              case info of
+                Just info' -> putStrLn $ "Implementing: " ++ getName xobj
```

Comment:
This currently prints a list, i.e. `:i Int.+` will print `Implements: (+)`, and if there are multiple it would be formatted as `(interface-a interface-b ...)`. To me, this is acceptable, but if we wanted we could make this a comma-separated list?